src/src/PluginStructs/P093_data_struct.cpp

```diff
@@ -362,6 +382,26 @@ void P093_data_struct::applySettings() {
   packet[21] = checkSum(packet, 21);
 
   sendPacket(packet, PACKET_LEN);
+
+  if (_writeStatus.isDirty(RemoteTemperature)) {
+    packet[5] |= 0x07;
+    if(_wantedSettings.remoteTemperature > 0) {
+      packet[6] |= 0x01;
+      _wantedSettings.remoteTemperature = _wantedSettings.remoteTemperature * 2;
+      _wantedSettings.remoteTemperature = round(_wantedSettings.remoteTemperature);
+      _wantedSettings.remoteTemperature = _wantedSettings.remoteTemperature / 2;
+      if (_tempMode) {        //units that don't support 0.5 increment 
+        packet[8] = static_cast<uint8_t>(_wantedSettings.remoteTemperature * 2.0f + 128.0f);
+      } else {                //units that do support 0.5 increment 
+        packet[7] = static_cast<uint8_t>(3.0f + ((_wantedSettings.remoteTemperature - 10.0f) * 2.0f));
+      }
+    }
+    else {
+      packet[6] = 0x00;
+      packet[8] |= 0x80; //MHK1 send 80, even though it could be 00, since ControlByte is 00
```

Comment:
The original code has a simple assignment operator here, not an OR-assign, though effectively it doesn't make a difference, best would be to use just `=`.

Reply:
ok, i wondered why in the whole code the or assignment was used... thats why i choose this on also for the modification..

Reply:
In this specific case it should be an assignment, the other places should keep `|=` as multiple settings can be sent in a single packet, and some set a different in that offset.